src/v/transform/logging/log_manager.cc

```diff
@@ -111,13 +112,23 @@ class flusher {
           local_bufs, MAX_CONCURRENCY, [this, &batches](auto& pr) {
               auto& [name, events] = pr;
               model::transform_name_view nv{name};
-              auto pid = _client->compute_output_partition(nv);
+              auto pid_res = _client->compute_output_partition(nv);
+              if (pid_res.has_error()) {
+                  vlog(
+                    tlg_log.debug,
```

Comment:
Should this be INFO or WARN?

Reply:
Sure. I like WARN for this - shouldn't be noisy assuming the topic exists